src/backend/marsha/core/tests/api/video/test_harvest_live.py

```diff
@@ -22,6 +23,86 @@ class VideoHarvestLiveAPITest(TestCase):
 
     maxDiff = None
 
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        cls.some_organization = factories.OrganizationFactory()
+        cls.some_video = factories.WebinarVideoFactory(
+            playlist__organization=cls.some_organization,
+            recording_slices=[
+                {"start": to_timestamp(timezone.now() - timedelta(minutes=10))},
+            ],
+            live_state=STOPPED,
+            live_info={
+                "medialive": {
+                    "input": {
+                        "id": "medialive_input_1",
+                        "endpoints": [
+                            "https://live_endpoint1",
+                            "https://live_endpoint2",
+                        ],
+                    },
+                    "channel": {"id": "medialive_channel_1"},
+                },
+                "mediapackage": {
+                    "id": "mediapackage_channel_1",
+                    "channel": {"id": "channel1"},
+                    "endpoints": {
+                        "hls": {
+                            "id": "endpoint1",
+                            "url": "https://channel_endpoint1/live.m3u8",
+                        },
+                    },
+                },
+            },
+        )
+
+    def assert_user_cannot_harvest_live(self, user, video):
+        """Assert the user cannot harvest the live."""
+
+        jwt_token = UserAccessTokenFactory(user=user)
+        response = self.client.post(
+            f"/api/videos/{video.pk}/harvest-live/",
+            HTTP_AUTHORIZATION=f"Bearer {jwt_token}",
+        )
+
+        self.assertEqual(response.status_code, 403)
+
+    @override_settings(LIVE_CHAT_ENABLED=True)
```

Comment:
as you are only testing the live_state, you can remove this and everything related to the chat like the `close_room` mock and the `mock_jwt_encode`